README.md

```diff
@@ -47,15 +47,12 @@ mvn clean install
 
 ## Developing
 
-The key concept is the `MachineLearningProvider`. To implement an OpenML provider, you have two options:
+The key concepts are described in the [openml-api](https://github.com/feedzai/feedzai-openml/tree/master/openml-api) module.
+See the [openml-example](https://github.com/feedzai/feedzai-openml/tree/master/openml-example) module for a trivial implementation.
```

Comment:
I think this is somehow duplicating the information we have a few lines above no?